src/promptflow/promptflow/_cli/_pf_azure/_flow.py

```diff
@@ -168,19 +206,21 @@ def add_parser_flow_download(subparsers):
 
 def dispatch_flow_commands(args: argparse.Namespace):
     if args.sub_action == "create":
-        create_flow(args)
+        create_or_update_flow(args)
```

Comment:
create and update behave the same?

Reply:
It was done because sdk interface `create_or_update` is the same for both usage. But it had other problems, after synced with Clement, we split it here again.